src/lib/pci_ahci.c

```diff
@@ -977,6 +985,7 @@ handle_identify(struct ahci_port *p, int slot, uint8_t *cfis)
 		buf[88] = 0x7f;
 		if (p->xfermode & ATA_UDMA0)
 			buf[88] |= (1 << ((p->xfermode & 7) + 8));
+		buf[93] = (1 | 1 <<14);
```

Comment:
The commit `Support TRIM on raw files via F_PUNCHHOLE` which does this is lacking a `cherry-picked from` note, could you add it please?